test/test_target_poses.yaml

```
@@ -0,0 +1,19 @@
+poses:
+  - position:
+      x: 1.0
+      y: 2.0
+      z: 3.0
+    orientation:
+      x: 0.0
+      y: 0.0
+      z: 0.0
+      w: 1.0
+  - position:
+      x: 4.0
+      y: 5.0
+      z: 6.0
+    orientation:
+      x: 0.0
+      y: 0.0
+      z: 0.0
+      w: 1.0
```

Changes/simplifies the format to align with the suggested YAML serialization changes

```suggestion
- x: 1.0
  y: 2.0
  z: 3.0
  qw: 1.0
  qx: 0.0
  qy: 0.0
  qz: 0.0
- x: 4.0
  y: 5.0
  z: 6.0
  qw: 1.0
  qx: 0.0
  qy: 0.0
  qz: 0.0
```